components/identities/identityCard.tsx

```diff
@@ -39,13 +42,15 @@ const IdentityCard: FunctionComponent<IdentityCardProps> = ({
     : `SID: ${tokenId}`;
   const [copied, setCopied] = useState(false);
   const [isHovered, setIsHovered] = useState(false);
-  const isMobile = useMediaQuery("(max-width:425px)");
+  const evmAddress = identity?.getUserDataWithField("evm-address") ?? undefined;
```

Comment:
I use the data from the indexer for this.`getUserDataWithField` is a function from the identity class that goes through indexed user data to get the right field. I made it like this, because at first I was querying different fields for each L2. I can update it to `identity.getEvmAddress()` so it's simpler, but I'm not sure to understand what's different than with other data, what are you referring to ?

Reply:
Oh I understood it wrong it's ok thx !